Headers/DebugServer2/Target/Linux/Thread.h

```diff
@@ -31,6 +31,9 @@ class Thread : public ds2::Target::POSIX::Thread {
 #endif
 
 protected:
+  void fillStoppointData();
```

Comment:
A breakpoint is a stoppoint though? So the name was intended to encompass both watchpoints and breakpoints.